public/js/main.js

```diff
@@ -3,7 +3,22 @@
 function tagElementClick() {
   this.classList.toggle("selected");
 
+  let config = JSON.parse(localStorage.getItem("config"));
```

Comment:
```suggestion
  let config = JSON.parse(localStorage.getItem("hidden-tags"));
```
Or something like that, what do you think ? It feel `config` is too generic